src/install/class-wordlift-install-service.php

```diff
@@ -108,6 +109,7 @@ public function __construct() {
 
 			// See #1621.
 			new Wordlift_Install_3_40_1(),
+			new Wordlift_Install_3_40_2(),
```

Comment:
Add a new line before `new Wordlift_Install_3_40_2()` or add a comment with the issue ID.